src/components/Dropdown/Dropdown.tsx

```diff
@@ -36,7 +36,7 @@ export class Dropdown extends React.PureComponent<DropdownProps & React.HTMLAttr
         this.toggleButton = React.createRef();
         this.handleClickOutside = this.handleClickOutside.bind(this);
         this.toggleDropdown = this.toggleDropdown.bind(this);
-        this.eventHandler = DropdownEvent.handler(this.handleClickOutside);
+        this.eventHandler = DropdownEvent.handler(this.handleClickOutside, DropdownEvent);
```

Comment:
How about we provide another version of this function in `DropdownEvent` which provides the second argument so the code that uses it does not have to?